R/derive_param_computed.R

```diff
@@ -149,12 +153,13 @@
 #'
 #' @export
 #'
-#' @examples
-#' library(tibble)
-#' library(dplyr)
-#' library(lubridate)
+#' @examplesx
+#'
+#' @caption Example 1 - Data setup
 #'
-#' # Example 1a: Derive MAP
+#' @info Examples 1a, 1b, and 1c use the following `advs` data.
```

Comment:
Minor: we've been capitalising the dataset in other examples
```suggestion
#' @info Examples 1a, 1b, and 1c use the following `ADVS` data.
```

Reply:
Thanks for pointing this out! I've made the change here and in the other datasets in Examples 2-4